modules/custom/social_path_manager/social_path_manager.module

```diff
@@ -0,0 +1,273 @@
+<?php
+
+/**
+ * @file
+ * The Social Path Manager module.
+ */
+
+use Drupal\Core\Entity\EntityInterface;
+use Drupal\Core\Form\FormStateInterface;
+use Drupal\Core\Url;
+use Drupal\group\Entity\Group;
+use Drupal\group\Entity\GroupType;
+
+/**
+ * Implements hook_form_alter().
+ */
+function social_path_manager_form_alter(&$form, FormStateInterface $form_state, $form_id) {
+  $social_group_types = _social_path_manager_group_types();
+
+  $group_forms = [];
+
+  foreach ($social_group_types as $social_group_type) {
+    $group_forms['edit'][] = "group_{$social_group_type}_edit_form";
+    $group_forms['add'][] = "group_{$social_group_type}_add_form";
+  }
+
+  if (in_array($form_id, $group_forms['add'], TRUE) || in_array($form_id, $group_forms['edit'], TRUE)) {
+    $form['path']['#type'] = 'fieldset';
+  }
+}
+
+/**
+ * Implements hook_module_implements_alter().
+ */
+function social_path_manager_module_implements_alter(&$implementations, $hook) {
+  if ($hook === 'form_alter') {
+    $group = $implementations['social_path_manager'];
+    unset($implementations['social_path_manager']);
+    $implementations['social_path_manager'] = $group;
+  }
+}
+
+/**
+ * Implements hook_entity_insert().
+ */
+function social_path_manager_entity_insert(EntityInterface $entity) {
+  _social_path_manager_update_alias($entity, 'create');
+}
+
+/**
+ * Implements hook_entity_update().
+ */
+function social_path_manager_entity_update(EntityInterface $entity) {
+  _social_path_manager_update_alias($entity, 'update');
+}
+
+/**
+ * Implements hook_entity_delete().
+ */
+function social_path_manager_entity_delete(EntityInterface $entity) {
+  _social_path_manager_update_alias($entity, 'delete');
+}
+
+/**
+ * Implements hook_batch_alter().
+ */
+function social_path_manager_batch_alter(&$batch) {
```

Comment:
This is used when using the bulk function to generate aliases.
admin/config/search/path/update_bulk
This will invoke when someone bulk updates the "group" paths.